Volume/Views/MainView/MainView.swift

```diff
@@ -56,6 +56,11 @@ struct MainView: View {
                 }
             }
         }
+        .onOpenURL { url in
+            if url.isDeeplink && url.host == ValidURLHost.article.toString() {
+                selectedTab = .home
```

Comment:
What happens in the case that a url isn't a deep link/isn't a valid URL host

Reply:
Do nothing. The app shouldn't interact with invalid URLs.